tests/src/test/resources/versions.conf

```diff
@@ -1,14 +1,17 @@
 versions {
-  intellij.build = "203.4203.26"
+  intellij {
+    build = "203.5784.10"
+  }
   plugins {
-    pythonCommunity = { id = "PythonCore", version = "203.4203.26" }
-    scala = { id = "org.intellij.scala", version = "2020.3.397", channel = "nightly" }
-    pants = {
+    pythonCommunity = { id = "PythonCore", version = ${versions.intellij.build} }
+    pythonUltimate  = { id = "Pythonid", version = ${versions.intellij.build} }
+    scala           = { id = "org.intellij.scala", version = "2020.3.442", channel = "nightly" }
```

Comment:
There is already a version [2020.3.628](https://plugins.jetbrains.com/plugin/1347-scala/versions/nightly).

Reply:
I tested with some newer version, it was failing. The purpose of this PR is mostly to merge all other changes. Testing versions can be done with automatic bump tool.